code/orphaned_procs/statistics.dm

```diff
@@ -63,7 +63,8 @@
 		laname = sanitizeSQL(H.lastattacker:real_name)
 		lakey = sanitizeSQL(H.lastattacker:key)
 	var/sqltime = time2text(world.realtime, "YYYY-MM-DD hh:mm:ss")
-	var/coord = "[H.x], [H.y], [H.z]"
+	var/url = "byond://[world.address]:[world.port]"
+	var/coord = "[H.x], [H.y], [H.z], [MAP_NAME], [url]"
```

Comment:
btw there's also a report_cyborg_death in the same file, might want to change that aswell